index.js

```diff
@@ -21,6 +21,9 @@ catch(err) {
  * @param {String} [options.electron] Path to custom electron version. If undefined
  *        will use environment variable `ELECTRON_PATH` or electron-prebuilt
  *        installed alongside.
+ * @param {String} [options.reporter=spec] Mocha reporter (non-debug mode only)
+ * @param {String} [options.reporterOptions] Additional options for the reporter
+ *        useful for specifying an output file if using the 'xunit' reporter.
```

Comment:
might be useful to add that this is expected in a querystring format, and that these options will vary based on the reporter